File: content/api-documentation/how-to/portfolio/examples/getPositions.js
const financefeast = require('@financefeasthq/financefeast-trade-api')
const financefeast = new financefeast()

// Get our position in AAPL.
aaplPosition = financefeast.getPosition('AAPL')

// Get a list of all of our positions.
financefeast.getPositions()
    .then((portfolio) => {
        // Print the quantity of shares for each position.
        portfolio.forEach(function (position) {
            console.log(`${position.qty} shares of ${position.symbol}`)
        })
    })

File: content/api-documentation/how-to/portfolio/examples/getPositions.go
package main

import (
	"fmt"
	"github.com/financefeasthq/financefeast-trade-api-go/financefeast"
)

func init() {
	financefeast.SetBaseUrl("https://paper-api.financefeast.markets")
}

func main() {
	// Get our position in AAPL.
	aapl_position, err := financefeast.GetPosition("AAPL")
	if err != nil {
		fmt.Println("No AAPL position.")
	} else {
		fmt.Printf("AAPL position: %v shares.\n", aapl_position.Qty)
	}

	// Get a list of all of our positions.
	positions, err := financefeast.ListPositions()
	if err != nil {
		fmt.Println("No positions found.")
	} else {
		// Print the quantity of shares for each position.
		for _, position := range positions {
			fmt.Printf("%v shares in %s", position.Qty, position.Symbol)
		}
	}
}